charts/all-in-one/templates/configmap-probe.yaml

```diff
@@ -1,46 +1,11 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-probe-script
-  namespace: {{ $.Release.Name }}
+  name: {{ .Chart.Name }}-probe-script
+  namespace: {{ $.Chart.Name }}
```

Comment:
how does it rendered? i guess it may be like `all-in-one-probe-script`??

Reply:
<img width="404" alt="image" src="https://github.com/planetarium/9c-infra/assets/30599098/5c8526df-91db-4ac1-83d3-82f087049609">
rendered `9c-network-probe-script`

Reply:
but I fixed it to use `{{ $.Release.Name }}`